test/components/hover-card/entity.spec.js

```diff
@@ -0,0 +1,80 @@
+import DlData from '../../../src/components/dl-data.vue';
+import HoverCardEntity from '../../../src/components/hover-card/entity.vue';
+
+import useHoverCardResources from '../../../src/request-data/hover-card';
+
+import { truncatesText } from '../../../src/util/dom';
+
+import testData from '../../data';
+import { mergeMountOptions, mount } from '../../util/lifecycle';
+import { testRequestData } from '../../util/request-data';
+
+const mountComponent = (options = undefined) =>
+  mount(HoverCardEntity, mergeMountOptions(options, {
+    container: {
+      requestData: testRequestData([useHoverCardResources], {
+        dataset: testData.extendedDatasets.last(),
+        entity: testData.standardEntities.last()
+      })
+    }
+  }));
+
+describe('HoverCardEntity', () => {
+  describe('entity property data', () => {
+    it('shows the first 5 properties', () => {
```

Comment:
This test and also the next test aren't super related to this PR. However, we don't currently have any tests of the entity hover card. (I plan to add more.) Since I'm adding a new test for this PR, I thought it made sense to add basic tests that the entity property data is being shown as expected.